lib/puppet-strings/yard/parsers/puppet/parser.rb

```diff
@@ -22,10 +22,12 @@ def initialize(source, filename)
   # @return [void]
   def parse
     begin
-      Puppet[:tasks] = true if Puppet.settings.include?(:tasks)
-      if Puppet::Util::Package.versioncmp(Puppet.version, "5.0.0") < 0 && @file.to_s.match(/^plans\//)
-        log.warn "Skipping #{@file}: Puppet Plans require Puppet 5 or greater."
-        return
+      if @file.to_s.match(/^plans\//)
```

Comment:
Would this work well enough? Original use case is there to only match `plans/` but an additional check for `/plans/` matches for the test without, i believe, hitting any accidental matches for something that isnt a plan?

```suggestion
      if @file.to_s.match(/[^plans|\/plans]\//)
```